prow/config/config.go

```diff
@@ -1379,10 +1384,14 @@ func ReadJobConfig(jobConfig string, yamlOpts ...yaml.JSONOpt) (JobConfig, error
 			}
 			return nil
 		}
-
 		if filepath.Ext(path) != ".yaml" && filepath.Ext(path) != ".yml" {
```

Comment:
understand that this is not from your PR, but I'm pretty sure that `.yml` files are not used by prow, do you know is this expected?

Reply:
Looks like this changed back in 2018 https://github.com/kubernetes/test-infra/pull/9137. I'm pretty sure both are considered valid extensions so its definitely better to respect both.

Reply:
ahh.. taking a deeper look and the problem was `config/jobs/**/*.yaml` at https://github.com/kubernetes/test-infra/blob/91011caf8dca5b240f34bd04a4091567cbe44e73/config/prow/plugins.yaml#L516, which is the reason why `.yml` files are not respected

Reply:
btw https://github.com/kubernetes/test-infra/pull/24675